sentinel-core/src/main/java/com/alibaba/csp/sentinel/slots/statistic/StatisticSlot.java

```diff
@@ -115,52 +115,31 @@ public void entry(Context context, ResourceWrapper resourceWrapper, DefaultNode
 
             throw e;
         } catch (Throwable e) {
-            // Unexpected error, set error to current entry.
+            // Unexpected internal error, set error to current entry.
             context.getCurEntry().setError(e);
 
-            // This should not happen.
-            node.increaseExceptionQps(count);
-            if (context.getCurEntry().getOriginNode() != null) {
-                context.getCurEntry().getOriginNode().increaseExceptionQps(count);
-            }
-
-            if (resourceWrapper.getEntryType() == EntryType.IN) {
-                Constants.ENTRY_NODE.increaseExceptionQps(count);
-            }
             throw e;
         }
```

Comment:
I found that with the new design, we needn't increaseExceptionQps here(Unexpected error), which looks nicer.
By the way, `context.getCurEntry().setError(e)` looks equivalent to `Tracer.traceContext(e, context)`, and the difference is that traceContext has `shouldTrace` judgement.
Since almost never happens here, both are okay, right?